MovieGallery/Pages/Search.cshtml

```diff
@@ -105,63 +105,76 @@
 
 else
 {
-    if (Model.SearchCompleted)
+    if (Model.CorrectYear)
```

Comment:
Validation added to see if the year entered is in a correct format and no future date has been entered. Suppose if we enter 10, it should give a message to enter the year in a valid format.